dell-r730xd/config.json

```diff
@@ -10,6 +10,12 @@
             "contains": "04N3DF"
         }
     ],
+    "skuConfig" : {
+        "redfish" : {
+            "computeType": "Rackmount"
+        },
+        "bmcMac": "rmm.data.MAC Address"
```

Comment:
Not sure where bmcMac is found on this Dell.  If it follows prior Dell art, then:

'bmcMac': 'bmc.data.MAC Address'